mods/parsoid.js

```diff
@@ -618,7 +614,7 @@ PSP.makeTransform = function(from, to) {
             var innerRes = res.body[to];
             innerRes.status = 200;
             // Handle bodyOnly flag
-            if (to === 'html' && req.body.bodyOnly) {
+            if (from === 'sections' && to === 'html' && req.body.bodyOnly) {
```

Comment:
Because you pass `bodyOnly` along in the request to Parsoid, and (in v3) it has already stripped the body tag from the output when you get it back.

But the `sections` stuff generates its own body tag around the output, which then has to be manually stripped here.  This is a bug, but like I said, I don't fully understand what the `sections` stuff is doing and could use some help.  I'm worried I could screw up the offsets if I just blindly remove `<body>` from the code which generates the sections output.